src/components/form/LabeledInput.vue

```diff
@@ -0,0 +1,167 @@
+<script>
```

Comment:
(I'm skipping reviewing the files imported from dashboard)

Reply:
I put them all in a separate initial commit for the benefit of the reviewer.

Reply:
Yep, saw that; it was appreciated. Just noting it here in case we ever want to look at that file again, so we can trace where it came from :)